### PR TITLE
Feature documentation

### PR DESCRIPTION
**Pull request by myronschippers**
Updates to project documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). Change ME!!!
```
Comment (scottbromander) [inline README.md:1]: Did this get changed!?!?!?!

File: README.md [hunk 2]
```diff
@@ -66,3 +66,7 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/de
 ### `npm run build` fails to minify
 
 This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+
+## Lecture notes
+
+I really like branching YAY!!!!
```
Comment (scottbromander) [inline README.md:72]: But do you, really!?

Comment (myronschippers) [reply README.md:72]: a little

**Comment by scottbromander** [comment]
This is good to go

**Pull request merged**